nixos/tests/all-tests.nix

```
@@ -581,8 +581,8 @@ in {
   netbird = handleTest ./netbird.nix {};
   neo4j = handleTest ./neo4j.nix {};
   netdata = handleTest ./netdata.nix {};
-  networking.networkd = handleTest ./networking.nix { networkd = true; };
-  networking.scripted = handleTest ./networking.nix { networkd = false; };
+  networking.networkd = handleTest ./networking/networkd-and-scripted.nix { networkd = false; };
+  networking.scripted = handleTest ./networking/networkd-and-scripted.nix { networkd = true; };
```

I think you accidentally switched the booleans.
```suggestion
  networking.networkd = handleTest ./networking/networkd-and-scripted.nix { networkd = true; };
  networking.scripted = handleTest ./networking/networkd-and-scripted.nix { networkd = false; };
```

---

oh yeah you right, fixed.